front-end/README.md

```diff
@@ -17,7 +17,7 @@ Create a `.env` file with the following environmen variable
 REACT_APP_HASURA_GRAPHQL_URL="http://localhost:8080/v1/graphql"
 REACT_APP_JWT_PUBLIC_KEY="-----BEGIN PUBLIC KEY-----\nMIIBI...wIDAQAB\n-----END PUBLIC KEY-----"
 REACT_APP_NETWORK="polkadot"
-REACT_APP_WS_PROVIDER="wss://rpc.polkadot.io" # ws://127.0.0.1:9944
```

Comment:
inline comments are not allowed in .env, so keeping it this way made the `yarn start` crash.